pkgs/development/python-modules/plink/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  sphinx,
+  tkinter,
+}:
+
+buildPythonPackage rec {
+  pname = "plink";
+  version = "2.4.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "3-manifolds";
+    repo = "plink";
+    tag = "${version}_as_released";
+    hash = "sha256-+WUyQvQY9Fx47GikzJ4gcCpSIjvk5756FP0bDdF6Ack=";
+  };
+
+  build-system = [
+    setuptools
+    sphinx
+  ];
+
+  dependencies = [ tkinter ];
+
+  pythonImportsCheck = [ "plink" ];
+
+  meta = {
+    description = "Full featured Tk-based knot and link editor";
+    homepage = "https://3-manifolds.github.io/PLink";
+    license = lib.licenses.gpl2Plus;
```

Comment:
The readme links to https://www.gnu.org/licenses/old-licenses/gpl-2.0.txt; how did you come to gpl2Plus?

Reply:
Ah, the `later version` part refers to v2+?